packages/migrate/src/commands/DbPull.ts

```diff
@@ -458,7 +458,8 @@ ${`Run ${chalk.green(getCommandWithExecutor('prisma generate'))} to generate Pri
           message += warning.affected.map((it) => `- Enum "${it.enm}", value: "${it.value}"`).join('\n')
         } else if (warning.code === 5 || warning.code === 6 || warning.code === 8) {
           message += warning.affected.map((it) => `- Model "${it.model}", field: "${it.field}"`).join('\n')
-        } else if (warning.code === 7 || warning.code === 14 || warning.code === 18 || warning.code === 19) {
+        } else if ([7, 14, 18, 19, 27, 30].includes(warning.code)) {
```

Comment:
This looks much nicer but makes the type of `warning.affected` below `any`, while explicit comparisons with each code allowed TS to narrow down the type.

Reply:
Yeah, I'm trying to figure out a way to not lose the typing information 🤔